js/civs.js

```diff
@@ -1,18 +1,214 @@
-function civ(name, tree) {
-    resetToDefault(tree);
+function parseCiv(civ) {
+   let builder = new CivBuilder({uniqueUnit: civ.unique[0],
+         uniqueUnitElite: civ.unique[1],
+         uniqueTechOne: civ.unique[2],
+         uniqueTechTwo: civ.unique[3],
+         monkPrefix: civ.monkPrefix});
 
-    let selectedCiv = civsConfig[name];
+   if (civ.disabled) {
+      if (civ.disabled.buildings) {
+         builder.disableBuildings(civ.disabled.buildings);
+      }
+      if (civ.disabled.units) {
+         builder.disableUnits(civ.disabled.units);
+      }
+      if (civ.disabled.techs) {
+         builder.disableTechs(civ.disabled.techs);
+      }
+   }
 
-    let enabled = selectedCiv.enabled || {};
-    let disabled = selectedCiv.disabled || {};
-    let uniqueConfig = selectedCiv.unique || {};
-    if (selectedCiv.disableHorses) {
-        disableHorses();
-    }
+   if (civ.enabled) {
+      if (civ.enabled.buildings) {
+         builder.enableBuildings(civ.enabled.buildings);
+      }
+      if (civ.enabled.units) {
+         builder.enableUnits(civ.enabled.units);
+      }
+      if (civ.enabled.techs) {
+         builder.enableTechs(civ.enabled.techs);
+      }
+   }
+
+   if (civ.disableHorses) builder.disableHorses();
+   
+   return builder.build();
+}
+
+function civ(name) {
+   let selectedCiv = parseCiv(civsConfig[name]);
+
+   SVG.select('.cross').each(function(i) {
+      if (SVGObjectIsAlreadyEnabled(this)) {
+         return;
+      }
+
+      let id = parseSVGObjectId(this.id());
+      if (!id) {
+         return;
+      }
+
+      if (selectedCiv.disabledUnits.includes(id) ||
+          selectedCiv.disabledBuildings.includes(id) ||
+          selectedCiv.disabledTechs.includes(id)) {
```

Comment:
Attention, tech ids and unit ids (buildings are technically units) are separate and both start from 0. It is just by chance that they do not overlap often. 
Taking away the Feudal Age (id=101) from mesoamerican civilisations just because they do not get access to stables (id=101) ist just mean though.
![grafik](https://user-images.githubusercontent.com/3027817/87248230-b3260180-c458-11ea-80dc-58c4d301113e.png)

Reply:
Ah yup, I had a feeling it was too good to be true. Changed back to the three `if`s, with spaced out brackets 😉